### PR TITLE
osd: add log when pg reg next scrub

### PR DESCRIPTION
**Pull request by dongbula**
Signed-off-by: lvshuhua <lvshuhua@cmss.chinamobile.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/osd/PG.cc [hunk 1]
```diff
@@ -4132,6 +4132,8 @@ void PG::reg_next_scrub()
 					       scrub_min_interval,
 					       scrub_max_interval,
 					       must);
+  dout(10) << __func__ << " pg " << pg_id << " register next scrub, scrub time "
+      << scrubber.scrub_reg_stamp << (must ? ", it is must" : "") << dendl;
```
Comment (xiexingguo) [inline src/osd/PG.cc]: `<< ", must = " << (int)must << dendl;` ?

Comment (dongbula) [reply src/osd/PG.cc]: good

**Comment by xiexingguo** [comment]
retest please

**Comment by xiexingguo** [comment]
retest this please

**Pull request merged**